src/components/CodeView/index.tsx

```diff
@@ -113,15 +113,26 @@ export class CodeViewBase extends React.Component<Props> {
     let slowAlert;
 
     if (
-      codeShouldBeTrimmed(content.length, _slowLoadingCharCount, isMinified)
+      codeShouldBeTrimmed({
+        codeCharLength: content.length,
+        codeLineLength: codeLines.length,
+        isMinified,
+        minifiedFileTrimmedCharCount: _minifiedFileTrimmedCharCount,
+        slowLoadingLineCount: _slowLoadingLineCount,
+      })
     ) {
       if (!shouldAllowSlowPages({ location })) {
-        codeLines = getLines(
-          `${content.substring(
-            0,
-            _trimmedCharCount,
-          )} /* truncated by code-manager */`,
-        );
+        if (isMinified) {
+          codeLines = getLines(
+            `${content.substring(
+              0,
+              _minifiedFileTrimmedCharCount,
+            )} /* truncated by code-manager */`,
```

Comment:
in DiffView this string is defined in a constant, `changeContentAddedByTrimmer`

Reply:
True. I should use that constant here. Thanks!

Reply:
https://github.com/mozilla/addons-code-manager/pull/1539/commits/7dd2501ce90886214d702bdf8900c9dc4d2fdde1